Akka.Persistence.Linq2Db.Benchmark.DockerComparisonTests/Akka.Persistence.Linq2Db.Benchmark.DockerComparisonTests.csproj

```diff
@@ -1,4 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
+    <Import Project="..\src\common.props" />
```

Comment:
We can skip explicitly referencing `common.props` if we just name the file `Directory.Build.props`, which is probably how we should do this going forward.... BUUUT dependabot may not like it.... So we'll stick with this for now.